authors/apps/articles/views.py

```diff
@@ -85,3 +86,145 @@ def destroy(self, request, pk=None):
         article.delete()
         return Response({"message": "article deleted successfully"},
                         status=status.HTTP_200_OK)
+
+
+class CommentViewSet(viewsets.ViewSet):
+    """Class to handle comments route'
+
+    One can post a comment, retrieve all, retrieve one, update,
+    delete comments
+    """
+    queryset = Comments.objects.all()
+    serializer_class = CommentSerializer
+    permission_classes = (IsAuthenticatedOrReadOnly,)
+    renderer_classes = (ArticleJsonRenderer,)
+
+    def get_specific_comment(self, article_id, comment_id, request):
+        """This methos a single comment related to a specific article"""
+        try:
+            Article.objects.get(pk=article_id)
```

Comment:
We'll need to abstract out the getting of common things e.g Articles in viewsets. You can leave it for now but change it in the code clean up.

Reply:
How do i abstract? Example in the articles scenario?

Reply:
You can abstract the process of getting an article and returning an error if it is not found. In the latest changes there is a function called get_article that can do that

Reply:
You mean the `get_article` in the serializer?

Reply:
I will look into it

Reply:
I have made the changes on the final rebase commit..